CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## Version 1.30.0 (2025-01-27)
+
 ## Version 1.30.0-RC (2025-01-27)
```

Comment:
hm, maybe we should have removed `## Version 1.30.0-RC` in main (also, I think we want `-rc.1`, or did I get it backwards?)